src/routes/study/study.controller.ts

```diff
@@ -251,30 +255,30 @@ const deleteStudy = async (req: Request, res: Response) => {
 
 const searchStudy = async (req: Request, res: Response) => {
   const keyword: string = req.query.keyword as string;
-  const frequencyFilter: string = req.query.frequency as string;
-  const weekdayFilter: string = req.query.weekday as string;
-  const locationFilter: string = req.query.location as string;
+  const frequencyFilter = req.query.frequency as FrequencyEnum;
+  const weekdayFilter = req.query.weekday as WeekDayEnum;
+  const locationFilter = req.query.location as LocationEnum;
```

Comment:
현재 enum 타입들로 assertion 해주고있는데 검증로직이 없어서 의도하지 않은 동작의 가능성이 있어 보입니다

```suggestion
  if (
    !Object.values(FrequencyEnum).includes(req.query.frequency) ||
    ... // WeekDayEnum, LocationEnum 도 동일
  ) throw new Error(BAD_REQUEST);
```
처럼 잘못된 쿼리파라미터 요청에 대해 400코드로 응답해도 좋을 것 같습니다

Reply:
오 감사합니다! 말씀해주신 방식이 훨씬 좋은 것 같아서 해당하는 것들 전부 수정하면 될 것 같네요.